packages/sdk/src/constants/constants.ts

```diff
@@ -1,18 +1,74 @@
-export enum Network {
+export enum Networks {
   Mainnet = 'mainnet',
   Staging = 'staging',
   Goerli = 'goerli',
   Kovan = 'kovan'
 }
 
-export enum Chain {
+export enum ChainIds {
+  Ethereum = 1,
+  Optimism = 10,
+  Arbitrum = 42161,
+  Polygon = 137,
+  Gnosis = 100,
+}
+
+export enum ChainNames {
+  Ethereum = 'Ethereum',
+  Optimism = 'Optimism',
+  Arbitrum = 'Arbitrum',
+  Polygon = 'Polygon',
+  Gnosis = 'Gnosis',
+}
+
+export enum ChainSlugs {
```

Comment:
Shouldn't enum names be singular as best practice?

https://stackoverflow.com/a/1336012/1439168

Reply:
i was debating this as well. the main reason for the plural naming was to minimize the number of duplicate name exports. this is now addressed in the latest commit